### DISCUSSION TITLE
The Default Tiddler Colour Setting

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#8408] [Ideas]
The variable `$:/config/DefaultTiddlerColour` sets the default color, which is primarily used for tag pills and icons. 

Can I call a procedure to get the palette default color and manipulate it (make a bit lighter or darker)?

**Comment by pmario** [comment]
`<<colour tag-background>>` will give you the value used in the palette. 

If you search for "tag" in the [PaletteManager Edition](https://wikilabs.github.io/editions/palette-manager/) you will get all tag-related colour settings

![image](https://github.com/user-attachments/assets/1c02a2d8-1752-4a5c-826d-625800b7dcc6)
